news/2024/2024-05-21-the-perennial-returns.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "The Perennial Returns"
+date: 2024-05-21 12:00:00 +0000
+---
+
+After a year-long hiatus we're brushing off the dust! The Perennial returns for its third iteration – and we've got more excitement than ever!
+
+![](/wiki/shared/news/2024-05-21-the-perennial-returns/Banner@2x.png)
+
+**[The Perennial](https://www.theperennial.net/)**, is a recurring open rank, international tournament using classic, polished formats and unique additions to bring excitement throughout the tournament. In 2021, the Perennial was one of very few international tournaments to ship trophy prizes across the world to the top three teams. In 2022, the Perennial was the first tournament in osu! to commission a song performed by a live band as an original track for the tournament. Further, the Perennial in 2022 became the second osu! Standard community tournament to ever receive the honor of being tri-badged, with the top three teams in the tournament all receiving profile badges for their accomplishment. This year promises to be better than either of our preceding iterations with more original tracks, competition just as tough, and some surprises along the way.
+
+## Original music and original maps
+
+The Perennial also has a history, albeit a short one, of bringing out the best in our mappers and staff. Many of the custom maps from both 2021 and 2022 are either ranked or loved, despite not being original songs, and many of those with leaderboards are now player favorites. Take [Nathan](https://osu.ppy.sh/users/4785223)'s map of [Angel Salad](https://osu.ppy.sh/beatmapsets/1471137#osu/3020221) by [katagiri](https://osu.ppy.sh/beatmaps/artists/176) as an example.
```

Comment:
instead of bringing out one example, it'd probably be better to provide a link to the a mappool

Reply:
Here are the archived mappools:

- 2021: https://docs.google.com/spreadsheets/d/1cFnN6DmM7QpzXta4ogYA6AepDD9jLrbA1sJ5HysirXk/edit?usp=sharing
- 2022: https://docs.google.com/spreadsheets/d/18Qx_9LOV580d5vzgdJkvzOyY3W_JTNN1vkgyZu70Duw/edit?usp=sharing